src/app/series/directives/series-podcast.component.html

```diff
@@ -117,21 +117,20 @@
       <input type="text" readonly [ngModel]="podcast?.publishedUrl" name="publishedUrl"/>
     </prx-fancy-field>
 
-    <prx-fancy-field label="Public Feed Url" class="feed-url">
-      <div *ngIf="podcast?.hasPublicFeed">
-        <div class="fancy-hint">
-          The public URL for your podcast feed. Feel free to share this URL with listeners. If you need to alter this URL, please
-          do so safely by using the New Feed Url field in the Advanced settings.
-        </div>
-        <input type="text" readonly [ngModel]="podcast?.publicFeedUrl" name="publicFeedUrl" #pubFeed/>
-        <button [publishCopyInput]="pubFeed">Copy</button>
-        <a class="button" target="_blank" rel="noopener" [href]="podcast?.publicFeedUrl">Open Link</a>
```

Comment:
oh, and it also is a little tricky  to add advance confirm into the mix -- doesn't play terribly well with publishCopyInput, which wants to have a manually added input field instead of the one given by the publish fancy field (which is the input field that the advance confirm watches for blurring)